.github/workflows/e2e.yaml

```diff
@@ -72,4 +72,8 @@ jobs:
           - "-X main.version=${{ github.sha }}"
         EOF
         docker run $(go run ./ build ./test/ --platform=${PLATFORM}) --wait=false 2>&1 | grep "${{ github.sha }}"
+        
+        # Check that using sbom-dir works.
+        KO_DOCKER_REPO=""  go run ./ build -t test --push=false --sbom-dir ./sbom-data ./test | grep "Writing SBOM to"
```

Comment:
If you `jq *.json` in the next line, you don't need to `grep` out the line. It might be nice to see all the output, and if writing fails or doesn't happen then the next line will fail anyway.